src/ui/public/vislib/lib/types/point_series.js

```diff
@@ -130,6 +130,13 @@ export function VislibTypesPointSeries() {
             }
           }
         ];
+      } else {
+        const categoryAxis1 = config.categoryAxes.find((categoryAxis) => {
+          return categoryAxis.id === 'CategoryAxis-1';
+        });
+        if (categoryAxis1) {
+          categoryAxis1.title.text = data.get('xAxisLabel');
```

Comment:
actually never from kibana (we don't expose an option to set category axis title), but if we would then this would override whatever setting we would put in there.
(vislib accepts category axis title text config ... and this would override it)